mcux/README

```diff
@@ -17,7 +17,15 @@ Origin:
 
    NXP MCUXpresso SDK Connectivity Framework
    https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-connectivity-framework
-   TAG: MCUX_2.15.000_RW610_RC1
+   TAG: MCUX_2.15.000_RW610_RC1 - MCUX_2.16.000(For connected_mcu platform files used by MCXW71)
```

Comment:
These changes should be placed with the commits that actually import the middlewares